src/main/java/org/ice4j/ice/DefaultNominator.java

```diff
@@ -314,18 +315,24 @@ public RelayedCandidateTask(CandidatePair pair)
          */
         public void propertyChange(PropertyChangeEvent evt)
         {
-            /* check list has running out of ordinary checks, see if all other
-             * candidates are FAILED, in which case we nominate immediately
-             * the relayed candidate
-             */
+            // Make it clear that PROPERTY_CHECK_LIST_CHECKS is in use here.
+            if (!CheckList.PROPERTY_CHECK_LIST_CHECKS.equals(
+                        evt.getPropertyName()))
+            {
+                return;
+            }
+
+            // check list has running out of ordinary checks, see if all other
```

Comment:
i know you just copy-pasted this, but looks like there is a typo?  is it "check list has **run** out of ordinary checks"?